app/config.py

```diff
@@ -360,6 +360,18 @@ class Config(object):
                 "schedule": crontab(hour=9, minute=0, day_of_week="mon"),
                 "options": {"queue": QueueNames.REPORTING},
             },
+            # first tuesday of every month
+            "change-dvla-api-key": {
+                "task": "change-dvla-api-key",
+                "schedule": crontab(hour=9, minute=0, day_of_week="tue", day_of_month="1-7"),
+                "options": {"queue": QueueNames.REPORTING},
```

Comment:
Very minor, but maybe `PERIODIC` is a better queue?

Reply:
yes absolutely should be and was my intention, sorry dont know how this got in

Reply:
fixed